charts/zipkin/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
what is different about most helm charts is even if they have a 'ci' directory, they don't have a test pod. So unless I am missing something, you only test that the thing didn't crash on `ct install`. This file allows you to write some things without moving to a more fancy framework like https://github.com/helm-unittest/helm-unittest (not sure that one is unit tests or integration, but anyway not using it yet)

Thanks @tamalsaha for pointing it out, in case we do use it one day.